src/docs/admin/references/ServiceNowReference.adoc

```diff
@@ -0,0 +1,560 @@
+*Product Name:* ServiceNow +
+*Product Version:* Quebec +
+*Compatibility:* Tokyo
+
+==== Overview
+
+The ServiceNow integration provides means to establish a bidirectional communication
+between a ProActive server, and a ServiceNow instance.
+
+==== Objectives
+
+The goal is to provide templates and coding elements to help ServiceNow and ProActive engineers to easily integrate both solutions in their pipelines.
+
+It boils down to: +
+
+* Provide reusable code in ServiceNow to query ProActive's REST APIs.
+* Provide workflows in ProActive's catalog to query ServiceNow's REST APIs.
+
+==== Installation
+
+This section describes the available processes to set up the required files in ServiceNow in order to communicate
+with ProActive. The process choice is dictated by the architecture the integration will have in ServiceNow. Because ServiceNow
+is application-based, the natural course of action is to create a dedicated ProActive application.
```

Comment:
These 2 phrases are hard to read and understand. "The process choice is dictated by the architecture the integration will have in ServiceNow", I think you're missing something between "the architecture" and "the integration".
Try not to start the second phrase with "Because", you can use : As the result, Being that, ... e.g. As the result of being an application-based component, the natural course of action for ServiceNow is to. create ...